apps/shell/Kconfig

```diff
@@ -65,10 +65,11 @@ config SECURED_TASH
 if SECURED_TASH
 
 config TASH_PASSWORD_SHA256
-	string "SHA256 encrypted password"
+	string "SHA256 encrypted password (32 Bytes)"
 	default "0000"
 	---help---
-		Enable password to enter TASH.
+		Enable password to enter TASH. The password is hashed by SHA256 so that
+		the length of this config should be 32 bytes (64 characters).
```

Comment:
it it fixed 32 byte or can be <= 32 bytes?

Reply:
fixed 32 bytes